extension/cart/handleCartErrors.js

```diff
@@ -18,7 +18,7 @@ module.exports = async (error, context, input) => {
     }
   }
 
-  if (error.code && error.code === 'ENOTFOUND') {
+  if (error.code && (error.code === 'ENOTFOUND' || error.code === 'EFORBIDDEN')) {
```

Comment:
@andre-kraus so we also create a new cart when it is forbidden to retrieve one? I can see a problem in this logic. Should it not retrieve a guest cart instead?